src/data/finset/basic.lean

```diff
@@ -186,7 +186,7 @@ instance decidable_mem' [decidable_eq α] (a : α) (s : finset α) :
 
 /-! ### extensionality -/
 theorem ext_iff {s₁ s₂ : finset α} : s₁ = s₂ ↔ ∀ a, a ∈ s₁ ↔ a ∈ s₂ :=
-val_inj.symm.trans $ nodup_ext s₁.2 s₂.2
+val_inj.symm.trans $ s₁.2.ext s₂.2
```

Comment:
Using field numbers in combination with dot notation makes this a bit too cryptic, so let's replace `.2` with `.nodup` throughout this file:
```suggestion
val_inj.symm.trans $ s₁.nodup.ext s₂.nodup
```